### DISCUSSION TITLE
What's problem with this script using execute?

### DISCUSSION THREAD

**Discussion by Tokubara** [hluk/CopyQ#2408] [Q&A]
My platform is windows 11.
I write this command to convert html I copy in Edge to org mode with pandoc. I've installed pandoc. 
```
[Command]
Command="
    copyq:
    var html = str(input())
    var res = execute('pandoc', '-f', 'html', '-t', 'org', '--wrap=none', null, stdinData=html)
    serverLog(res.stdout)
    setData('text/plain', res.stdout)"
Icon=\xf042
InMenu=true
Input=text/html
Name=html to org
```
But it doesn't work, no error notifications but I don't get the result. I think copyq may not find pandoc, so I try absolute path, I do get error notifications, `Expected token )`.
```
var res = execute('C:\\Users\\12708\\AppData\\Local\\Pandoc\\pandoc.exe', '-f', 'html', '-t', 'org', '--wrap=none', null, stdinData=html)
```
But I cannot see why it expects `)`.
How can I make it work?

**Comment by hluk** [comment]
Try to use `/` as path separator ('`C:/Users/...'`). `\` is a special character used for escape sequences.

**Reply by Tokubara** [reply]
Thank you! Now it doesn't throw an error, but it doesn't work. I can't see why.

**Reply by hluk** [reply]
You may also need to call `removeData(mimeHtml)` at the end to drop the HTML format from the item so it does show only the new modified plain text.
